client/src/components/profile-box.js

```diff
@@ -26,9 +28,10 @@ const ProfileBox = ({
   let onUpdate = async e => {
     e.preventDefault();
     validateFields(async (err, values) => {
+      let validValues = pickby(values, identity);
```

Comment:
🤔 emm, it looks like an empty string is also considered as falsey.

If user tries to delete its bio instead of changing it, it won't work as what the user expects.

Maybe try?

```js
pickBy(object, val => (val !== null && val !== undefined));
```